core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -234,6 +234,7 @@ public void applyTo(TableMetadata.Builder metadataBuilder) {
 
   class SetSnapshotRef implements MetadataUpdate {
     private final String name;
+    // TODO - this can be primitive.
     private final Long snapshotId;
```

Comment:
TODO / possible follow up - This can be made primitive

Reply:
This TODO is no longer written in the code, but doesn't show as outdated because the comment is on the line below.